dm/tests/checkpoint_transaction/run.sh

```diff
@@ -41,8 +93,13 @@ function run() {
 		"\"stage\": \"Paused\"" 1
 	# check the point is the middle of checkpoint
 	num=$(grep "not receive xid job yet" $WORK_DIR/worker1/log/dm-worker.log | wc -l)
-	[[ $num -gt 0 ]]
-	sed -e '/not receive xid job yet/d' $WORK_DIR/worker1/log/dm-worker.log >$WORK_DIR/worker1/log/dm-worker.log
```

Comment:
where is this line?

Reply:
this line is deleted because if muti thread is write same log file, the log content can become confusing, so stop the dm-worker and delete whole dm-worker dir in https://github.com/pingcap/tiflow/pull/4035/commits/76180db7ea6f8b62a47929c730a7c02425d3e3c8